modules/auxiliary/scanner/http/billquick_txtid_sqli.rb

```diff
@@ -0,0 +1,195 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
Does `AutoCheck` work with scanners?
Also, what do we get from this being a `scanner` module rather than say, a `gather` module?

Reply:
`check` doesn't work, but AutoCheck kicks the check before the module runs.... so lets go with yes